track_history/tracker.py

```diff
@@ -59,8 +64,14 @@ def store_current_state(self, record_type=None):
         instance_deleted: bool = (
             record_type == self.history_record_model.RECORD_TYPES.deleted
         )
+        instance_undeleted: bool = (
```

Comment:
H: Looks like the `instance_undeleted` variable is not used, so it's probably not needed

Reply:
Good catch, removed.